source/_integrations/pushover.markdown

```diff
@@ -65,6 +65,7 @@ To use the highest priority, which repeats the notification every x seconds (`re
     target:
       - iphone11pro
     data:
+      push:
```

Comment:
Unfortunately, this is not correct.

This parameter doesn't exists and isn't fetched from the configuration.
See:

https://github.com/home-assistant/core/blob/c1945211fa4851a0e925f3853a9e04572509597e/homeassistant/components/pushover/notify.py#L69-L78

../Frenck